Src/Fido2/Fido2NetLib.cs

```diff
@@ -140,11 +144,11 @@ public CredentialMakeResult(string status, string errorMessage, AttestationVerif
     /// </summary>
     /// <param name="credentialIdUserParams"></param>
     /// <returns></returns>
-    public delegate Task<bool> IsCredentialIdUniqueToUserAsyncDelegate(IsCredentialIdUniqueToUserParams credentialIdUserParams);
+    public delegate Task<bool> IsCredentialIdUniqueToUserAsyncDelegate(IsCredentialIdUniqueToUserParams credentialIdUserParams, CancellationToken cancellationToken);
     /// <summary>
     /// Callback function used to validate that the Userhandle is indeed owned of the CrendetialId
     /// </summary>
     /// <param name="credentialIdUserHandleParams"></param>
     /// <returns></returns>
-    public delegate Task<bool> IsUserHandleOwnerOfCredentialIdAsync(IsUserHandleOwnerOfCredentialIdParams credentialIdUserHandleParams);
+    public delegate Task<bool> IsUserHandleOwnerOfCredentialIdAsync(IsUserHandleOwnerOfCredentialIdParams credentialIdUserHandleParams, CancellationToken cancellationToken);
```

Comment:
Since this is an public API we want to make the CancellationToken optional.

Reply:
Delegates are public but without intention to be called by a consumer. Consumers need to define the new argument anyway. The gain of not specifying =default is a compile check for forgotten cancellationToken.

__
That is also the reason why I did not add =default to implementations in the first place. Consumers should mainly access through an interface (opinionated). And library implementation itself has a compile check when calling methods inside implementation (method A calling B in the same class).

Reply:
> Delegates are public but without intention to be called by a consumer.

Alright, I see your point on the delegates.